puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/tests/testForeman.groovy

```diff
@@ -0,0 +1,135 @@
+node('fast') {
+
+    stage('Setup Git Repos') {
+
+        git url: "https://github.com/theforeman/foreman", branch: "develop"
+
+    }
+
+    stage("Setup RVM") {
+
+        withRVM([
+            'rvm gemset empty --force',
+            'gem update --no-ri --no-rdoc',
+            'gem install bundler --no-ri --no-rdoc'
+        ])
+
+    }
+
+    stage('Configure Environment') {
+
+        try {
+            sh "cp config/settings.yaml.example config/settings.yaml"
+            sh "sed -i 's/:locations_enabled: false/:locations_enabled: true/' config/settings.yaml"
+            sh "sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/settings.yaml"
+
+            sh "cp $HOME/postgresql.db.yaml config/database.yml"
```

Comment:
I meant this line @ehelms. Copying from $HOME enables changes to `$HOME/postgresql.db.yaml` without raising a PR or review. I think it would be better to instead define the contents of `config/database.yml` in here.

Reply:
Because, the part below this is I think entirely ugly and something like ERB or some ansible would do wonders.